Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -129,16 +133,28 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
         this.needsLayout();
     }
 
+    get variablesGroupType() { return this._variablesGroupType; }
+
+    get isEmpty() {return !this._propertyViews.length; }
```

Comment:
Style: this is not really a simple getter (i.e. it returns a member variable of the same name), so please expand this onto multiple lines